app/Console/Commands/CleanupNames.php

```
@@ -229,15 +201,8 @@ protected function createBackup(): void
             );
 
             exec($command, $output, $returnCode);
```

PHPMD (2.15.0)

Avoid unused local variables such as '$output'. (undefined)

(UnusedLocalVariable)